test/plugin.test.js

```diff
@@ -206,6 +211,20 @@ QUnit.test('includes the video name if provided', function(assert) {
         'wrote a placeholder for the name');
 });
 
+QUnit.test('includes custom data attribute if provided', function(assert) {
+  this.player.playlist(playlist);
+  this.player.playlistUi();
+
+  const items = this.fixture.querySelectorAll('.vjs-playlist-item');
+
+  assert.strictEqual(items[0].dataset.id,
```

Comment:
It would probably be good to check the actual values of these here as well. Seems like if they were both undefined this would pass, even though it should have a value.

Reply:
Added add'l assertions for that. Not shown inline here. Good call. Thx.